website/screens/overview/introduction/IntroductionPage.tsx

```diff
@@ -185,6 +185,47 @@ const sections = [
       </>
     ),
   },
+  {
+    title: "Documentation of other releases",
+    content: (
+      <>
+        <DxcParagraph>
+          This site is new and comes to replace the existing one. However, the
+          old documentation is still available through the following links for
+          consultation:
+        </DxcParagraph>
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            <DxcLink href="https://developer.dxc.com/tools/react" newWindow>
+              React CDK
+            </DxcLink>
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <DxcLink href="https://developer.dxc.com/tools/angular" newWindow>
+              Angular CDK
+            </DxcLink>
+          </DxcBulletedList.Item>
+        </DxcBulletedList>
+        <DxcParagraph>
+          <em>
+            Please note that these sites are discontinued and may not be updated
+            anymore.
+          </em>
+        </DxcParagraph>
+        <DxcParagraph>
+          The last development site can be accessed through this{" "}
```

Comment:
Rephrase both "last" appearances to "latest". _Last_ is used for something that happens at the end, when we are referring to the most recent version of something we should use _latest_.

Reply:
Done!